examples/neb_with_restart.lua

```diff
@@ -0,0 +1,347 @@
+--[[
```

Comment:
It is unclear why you have `neb_with_restart.lua` and `neb_with_restart-new.lua`, stick with one of them.